docusaurus/website/static/css/custom.css

```diff
@@ -9,6 +9,12 @@
 .homeContainer .button {
   background: #24292e;
   border-color: white;
+  transition: transform 150ms;
```

Comment:
```suggestion
  transition: background 0.3s, color 0.3s;
```

Sorry for the nit, let's stay consistent with the other buttons, see: https://github.com/facebook/Docusaurus/blob/b6aa6dac89203012ea42068a2ffe2e7b25e8f15d/packages/docusaurus-1.x/lib/static/css/main.css#L442